src/main/java/frc/robot/Commands/Climb.java

```diff
@@ -1,4 +1,23 @@
 package frc.robot.Commands;
 
-public class Climb {
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Subsystems.ClimberWinch;
+
+public class Climb extends SequentialCommandGroup {
+
+    public Climb() {
+        ClimberWinch climberWinch = ClimberWinch.getInstance();
+        addRequirements(climberWinch);
+        addCommands(
+                new MoveGenericSubsystem(climberWinch, climberWinch.getUpSpeed()),
```

Comment:
this won't work, like, physically.
you need to move closer to the bar after you open the telescopic arm for this system to catch it.